odl/space/fspace.py

```diff
@@ -503,6 +502,11 @@ def product(arg):
             return x_old(arg) * y_old(arg)
         y._function = product
 
+    def _inner(self, x1, x2):
```

Comment:
:+1: To also be honest, I simply copied it and prettified the message a little, at little gain.